doc/man1/ts.pod

```diff
@@ -242,7 +242,7 @@ The name of the file containing a DER encoded timestamp request. (Optional)
 =item B<-passin> password_src
 
 Specifies the password source for the private key of the TSA. See
-B<PASS PHRASE ARGUMENTS> in L<openssl(1)>. (Optional)
+the B<PASS PHRASE ARGUMENTS> in L<openssl(1)>. (Optional)
```

Comment:
Forgot to change this?

Reply:
Kinda...  my sed script wasn't calibrated for this variation, and I missed that it had missed this one.  Fixed